modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/dsl/JDBC.scala

```diff
@@ -74,8 +74,12 @@ object JDBC {
 
   def apply[F[_]](implicit ev: JDBC[F]): JDBC[F] = ev
 
-  def log[F[_]: Logging](e: Throwable, d: RetryDetails): F[Unit] =
-    Logging[F].error(s"Cannot acquire connection: ${e.getMessage}. ${retriesMessage(d)}")
+  def log[F[_]: Logging](e: Throwable, d: RetryDetails): F[Unit] = {
+    if (d.givingUp)
+      Logging[F].error(s"Cannot acquire connection: ${e.getMessage}. ${retriesMessage(d)}")
+    else
+      Logging[F].info(s"Warning. Cannot acquire connection: ${e.getMessage}. ${retriesMessage(d)}")
+  }
```

Comment:
Should be part of https://github.com/snowplow/snowplow-rdb-loader/pull/460/commits/9824bd39549484fa7d56f9e1ce69c82d28b056dc

Reply:
Agreed.